packages/legend-query-builder/src/components/__tests__/QueryBuilderParametersPanel.test.tsx

```diff
@@ -107,7 +172,7 @@ test(
 
 test(
   integrationTest(
-    'Query builder shows validation error for constant name if existing duplicate parameter name',
```

Comment:
There were a bunch of tests related to the constants panel in this file, so I moved them from here to `QueryBuilderConstantsPanel.test.tsx`